python/mrtrix3/commands/5ttgen/odf.py

```diff
@@ -0,0 +1,95 @@
+# HSVS SCRIPT
+# Copyright (c) 2008-2024 the MRtrix3 contributors.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Covered Software is provided under this License on an "as is"
+# basis, without warranty of any kind, either expressed, implied, or
+# statutory, including, without limitation, warranties that the
+# Covered Software is free of defects, merchantable, fit for a
+# particular purpose or non-infringing.
+# See the Mozilla Public License v. 2.0 for more details.
+#
+# For more details, see http://www.mrtrix.org/.
+
+import os
+import subprocess
+from mrtrix3 import MRtrixError
+from mrtrix3 import app, path, run
+
+
+def usage(base_parser, subparsers):  # pylint: disable=unused-variable
+    parser = subparsers.add_parser('odf', parents=[base_parser])
+    parser.set_author("Arkiev D'Souza (arkiev.dsouza@sydney.edu.au) & Robert E. Smith (robert.smith@florey.edu.au)")
+    parser.set_synopsis('Generate a 5TT image from ODF images.')
+    
+    parser.add_argument('template_odf_wm', type=app.Parser.ImageIn(), help='The input white-matter ODF')
+    parser.add_argument('template_odf_gm', type=app.Parser.ImageIn(), help='The input grey-matter ODF')
+    parser.add_argument('template_odf_csf', type=app.Parser.ImageIn(), help='The input cerebrospinal fluid ODF')
```

Comment:
```suggestion
    parser.add_argument('odf_wm', type=app.Parser.ImageIn(), help='The input white-matter ODF')
    parser.add_argument('odf_gm', type=app.Parser.ImageIn(), help='The input grey-matter ODF')
    parser.add_argument('odf_csf', type=app.Parser.ImageIn(), help='The input cerebrospinal fluid ODF')
```

Algorithm is not exclusively applicable to template ODFs; should be at least possible to run on subgle-subject data.